src/Logging.AzureAppServices/src/AzureAppServicesLoggerFactoryExtensions.cs

```diff
@@ -17,6 +17,22 @@ namespace Microsoft.Extensions.Logging
     /// </summary>
     public static class AzureAppServicesLoggerFactoryExtensions
     {
+        internal static string CustomPrefix;
```

Comment:
This doesn't work because multiple independent calls in the same assembly will cause this to be overwritten.

Reply:
This assembly is designed to be called only once when the IHostBuilder container is configured in Program.cs of a web application. Please refer to the next comment.

Reply:
The concept of Logging Provider is that it is called once throughout the application lifetime. It's either with CustomPrefix or without. And the only place this value is set is at the entry point where the provider is injected to IHostBuilder ConfigureLogging. Nowehere else. Please re-review this change. Thank you. - cc: @BrennanConroy